File: src/main/java/br/com/rfasioli/micrometermetrics/persistence/repository/PersonRepository.java
package br.com.rfasioli.micrometermetrics.persistence.repository;

import br.com.rfasioli.micrometermetrics.persistence.document.Person;
import org.bson.types.ObjectId;
import org.springframework.data.mongodb.repository.MongoRepository;

public interface PersonRepository extends MongoRepository<Person, ObjectId> {
}

File: src/main/kotlin/br/com/rfasioli/micrometermetrics/persistence/repository/PersonRepository.kt
package br.com.rfasioli.micrometermetrics.persistence.repository

import br.com.rfasioli.micrometermetrics.persistence.document.Person
import org.bson.types.ObjectId
import org.springframework.data.mongodb.repository.MongoRepository

interface PersonRepository : MongoRepository<Person, ObjectId>
